build.sh

```diff
@@ -20,7 +20,7 @@ fi
 if [ -z "$1" -o -z "$2" ]; then
   echo "Usage: $0 <godot branch> <mono version> [<mono branch> <mono commit hash>]"
   echo
-  echo "Examples: $0 3.1 mono-5.18.1.3"
+  echo "Examples: $0 3.2 mono-6.12.0.114 2020-02"
```

Comment:
```suggestion
  echo "Examples: $0 3.2 mono-6.12.0.114"
```
As mentioned above, the branch is not needed.

Reply:
I have updated as per your comments. Cheers.